docs/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+
+
+:zap: Please ensure that you reviewed the [guidelines for contributing](/CONTRIBUTING.md) to this repository. 
+
+
+What does this pull request contain? Please explain your changes.
+---------------------------------------------------
+* ...
+
+* ...
+
+* ...
+
+Does this close any currently open issue(s)?
+------------------------------------------
+…
```

Comment:
Please use the GitHub tooling for this by making use of the [right keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue),